test/puppetlabs/comidi_test.clj

```diff
@@ -271,14 +326,26 @@
 
     (handler {:uri "/bar/something" :request-method :get})
     (is (= (-> test-atom deref :route-info :route-id) "bar-:bar"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/false" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "false"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/true" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "true"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/wat" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "*"))
```

Comment:
"false" support seems kind of odd but it felt necessary to balance out the "true" changes. I don't think it can be hit either.